packages/generator/generators/draft/templates/README.txt

```diff
@@ -0,0 +1,27 @@
+---
+title: "<%= componentName %>"
+navTitle: "<%= componentName %>"
+summaryParagraph: <%= componentName %>s are a neat component
+tags: [""]
+demoStoryId:  <%= componentName %>-react--default-kaizen-site-demo
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+<WhenToUseAndWhenNotToUse>
+
+<WhenToUse>
+* Use this for the thing
+</WhenToUse>
+
+<WhenNotToUse>
+* Don't use this for the other thing though! 
+</WhenNotToUse>
+</WhenToUseAndWhenNotToUse>
+
+## See also
+
+## External links
+
```

Comment:
As above, perhaps some examples:

```
- [Polaris: button](https://polaris.shopify.com/components/actions/button)
- [Atlassian: button](https://www.atlassian.design/components/button/examples)
- [Spectrum: button](https://spectrum.adobe.com/page/button/)
- [Lightning: button](https://www.lightningdesignsystem.com/components/buttons/#site-main-content)
```

… although there is a risk they won't be updated

Reply:
Will `TODOs` suffice?